cleanlab/internal/label_quality_utils.py

```diff
@@ -15,30 +15,30 @@
 # along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
 
 """Helper functions for computing label quality scores"""
-
 import numpy as np
+from typing import Optional
 from cleanlab.count import get_confident_thresholds
 
 
 def _subtract_confident_thresholds(
-    labels: np.ndarray,
+    labels: Optional[np.ndarray],
     pred_probs: np.ndarray,
+    confident_thresholds: Optional[np.ndarray] = None,
```

Comment:
It's generally safer to add this extra arg to existing helper functions at the end of the arg-list, in case these helper functions are being used with positional arguments in the codebase.

Reply:
Makes a lot of sense!